### PR TITLE
Update Entrepreneurship

### PR DESCRIPTION
**Pull request by imskr**
issue #102

### CODE REVIEW

**Comment by imskr** [comment]
@alodahl @manaswinidas

File: roles-in-tech/entrepreneurship.md [hunk 1]
```diff
@@ -2,90 +2,88 @@
...
+    Extrinsic motivations often work against entrepreneurs by leading them to set unrealistic and overwhelming goals.
+
+- **Find personal meaning from building a business** 
+    In book, `The Art of The Start 2.0` Guy Kawasaki exhorts entrepreneurs 
+    to focus on  making meaning, not money. He has said many times that if your vision for your company is to grow it 
+    just to flip it to a large company or to take it public and cash out, "you're doomed.” Do it for meaning.
+    
+- **Satisfaction of doing something great** 
+    `Steve Jobs` summarized his intrinsic motivation 
+    in 2005 at Stanford in a talk titled `How to Live Before You Die` He said,
```
Comment (manaswinidas) [inline roles-in-tech/entrepreneurship.md]: Missing full stop.

Comment (imskr) [reply roles-in-tech/entrepreneurship.md]: Thanks for the review.

**Comment by manaswinidas** [comment]
@imskr please resolve the conflicts

**Comment by imskr** [comment]
Done @alodahl @manaswinidas

**Comment by imskr** [comment]
Please check @alodahl

**Pull request merged**